bibangamba/githubusers_react_native
# #166258383 build a home screen

#### What does this PR do?
adds a home screen as a list of `Lagos java developers`

#### Description of tasks to be done?
- add apollo client implementation
- create and design list item
- add `react-native-dotenv` to manage environment variables
- populate list with data from api
- add navigation from list to a mock details screen
- write tests for the implemented functionality

#### How can this be manually tested?
- clone the repo and check out the branch
- run the application with `npm start`
- send the application to an emulator by typing `a`(android) or `i`(iOS)
- the application should open up on a login screen 
- entering good credentials will take you to a single screen a `loading...` text
- after a few seconds, a list of developers should show up
- clicking on any developer should take you to a screen with the GitHub logo centred

#### Screenshots
![demo gif](http://g.recordit.co/ltiSARfyMW.gif)

#### What are the relevant pivotal tracker stories?
[#166258383](https://www.pivotaltracker.com/story/show/166258383)
